scss/_patterns_form-password-toggle.scss

```diff
@@ -11,4 +11,14 @@
     display: inline-block;
     margin-right: $sp-xx-small;
   }
+
+  // Override the input text so that it is more easy to read when it
+  // has been toggled to display the text. This expects that the input
+  // immediately follows the .p-form-password-toggle element.
+  .p-form-password-toggle + [type='text'] {
+    &[readonly],
+    &[readonly='readonly'] {
```

Comment:
Isn't the previous line `&[readonly]` enough to cover the cases where readonly attribute is present with any value?

Reply:
In my quick tests it did seem to work with just `[readonly]` but the base forms target both so I thought I better include both here so that there weren't any potential specificity clashes: https://github.com/canonical/vanilla-framework/blob/main/scss/_base_forms.scss#L52.

I'm happy to change that to `[readonly]` if you think that'd be fine.